KOF/Game/Models/Game.cs

```diff
@@ -0,0 +1,9 @@
+namespace KOTGame.Models;
+
+public class Game
+{
+    public int Id { get; set; }
+    public string? TokyoBoard { get; set; }
+    public int NumberOfPlayers { get; set; }
```

Comment:
I'm about to create the Board class and include the Players (each instance of Player class) so after that you're gonna be able to create an instance of it and replace this two lines 🙌🏻